assets/stylesheets/components/_forms.scss

```diff
@@ -160,29 +160,31 @@ input[type="checkbox"]:disabled {
 }
 
 .input-group {
-  input {
-    padding: 0 10px;
+  position: relative;
+  display: block;
+
+  input.form-control {
+    padding: 0 10px 0 48px;
+    float: none;
   }
 
   .input-group-addon {
-    background: $input-group-addon-bg;
-    border-radius: $input-border-radius;
+    position: absolute;
+    display: flex;
+    height: 100%;
+    margin-left: 16px;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+    z-index: 4;
     font-size: 1.125rem;
+    background: transparent;
```

Comment:
Why not use the `$input-group-addon-bg` variable here instead of hard-coding `transparent`?